### PR TITLE
[#120] Added e2e tests for flower and reuters classifier

### LINKED ISSUE #120: New test for flower_classider model
We have to add a new GPU tag because this model must be only run on a cluster with GPU support.
Additionally, we should refactor the e2e tests so that they use the yaml odahu files from the odahu-examples repo.

### PR DESCRIPTION
**Pull request by mcdoker18**
Removed all hardcoded e2e manifest files. For now these files are
downloaded from the examples git repository before tests.

This closes #120

### CODE REVIEW

File: packages/robot/odahuflow/robot/profiler_loader.py [hunk 1]
```diff
@@ -67,6 +68,7 @@ def get_variables(profile=None) -> typing.Dict[str, str]:
                 'MLFLOW_URL': os.getenv('MLFLOW_URL', f'https://{host_base_domain}/mlflow'),
                 # TODO: Remove after implementation of the issue https://github.com/legion-platform/legion/issues/1008
```
Comment (karbyshevds) [inline packages/robot/odahuflow/robot/profiler_loader.py]: is it still an actual issue?

Comment (mcdoker18) [reply packages/robot/odahuflow/robot/profiler_loader.py]: Yeap, @vlad-tokarev  will fix it soon.

File: packages/robot/odahuflow/robot/libraries/examples_loader.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#
+from os import makedirs
+from pathlib import Path
+
+import requests
+
+
+class ExamplesLoader:
+
+    def __init__(self, examples_http_url, examples_version) -> None:
```
Comment (vlad-tokarev) [inline packages/robot/odahuflow/robot/libraries/examples_loader.py]: type annotation?

Comment (mcdoker18) [reply packages/robot/odahuflow/robot/libraries/examples_loader.py]: Fixed

**Pull request merged**